bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -196,11 +196,11 @@ void rollLog(LastLogMark lastMark) throws NoWritableLedgerDirException {
             // which is safe since records before lastMark have been
             // persisted to disk (both index & entry logger)
             lastMark.getCurMark().writeLogMark(bb);
-            if (LOG.isDebugEnabled()) {
-                LOG.debug("RollLog to persist last marked log : {}", lastMark.getCurMark());
-            }
+
+            LOG.info("RollLog to persist last marked log : {}", lastMark.getCurMark());
+
             List<File> writableLedgerDirs = ledgerDirsManager
-                    .getWritableLedgerDirs();
+                    .getWritableLedgerDirsForNewLog();
```

Comment:
I think this is crucial fix. May be reflect in the commit summary?

Reply:
Done.